notebooks/HMM_Building.ipynb

```diff
@@ -86,12 +78,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 1,
```

Comment:
Looks like you need to import numpy

Reply:
done (I think we'd left it blank intentionally to prompt student input but agreed it was causing a weird error popup)